packages/monaco-graphql/src/api.ts

```diff
@@ -0,0 +1,19 @@
+import * as languageFeatures from './languageFeatures';
+import type { DocumentNode } from 'graphql';
+import { SchemaResponse } from 'graphql-languageservice';
+
+export class MonacoGraphQLApi {
+  private worker: languageFeatures.WorkerAccessor;
+  constructor({ accessor }: { accessor: languageFeatures.WorkerAccessor }) {
+    this.worker = accessor;
+  }
+
+  async getSchema(): Promise<SchemaResponse> {
```

Comment:
I don't think `this` is properly bound here (got an error when I had it linked)
```suggestion
  getSchema = async () => {
```

Reply:
Oh! I just remembered, that was because I called it from a global variable in the dev console 😆 Consider this a nit, haha

Reply:
ah! it's still a good pattern